sdk/lib/rtl/amd64/except.c

```diff
@@ -0,0 +1,117 @@
+/*
+* COPYRIGHT:         See COPYING in the top level directory
+* PROJECT:           ReactOS Run-Time Library
+* PURPOSE:           User-mode exception support for AMD64
+* FILE:              lib/rtl/amd64/except.c
+* PROGRAMERS:        Timo Kreuzer (timo.kreuzer@reactos.org)
+*/
```

Comment:
```suggestion
/*
 * PROJECT:     ReactOS Run-Time Library
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     User-mode exception support for AMD64
 * COPYRIGHT:   Copyright 2021 Timo Kreuzer <timo.kreuzer@reactos.org>
 */
```
According to the latest https://reactos.org/wiki/Coding_Style